xrmocap/core/evaluation/metric_manager.py

```diff
@@ -0,0 +1,118 @@
+# yapf: disable
+import logging
+from typing import List, Union
+from xrprimer.utils.log_utils import get_logger
+
+from .metrics.base_metric import BaseMetric
+from .metrics.builder import build_metric
+
+# yapf: enable
+
+
+class MetricManager:
+    """MetricManager is a class for multiple metrics evaluation.
+
+    It sorts metrics by ranks(descend), and call the calculation functions in
```

Comment:
Is it better to calculate metrics in ascending order, in which rank 0 metrics are the most fundamental metrics? It might be more convenient to add higher-level metrics in the future.

Reply:
Good idea. Fixed in [c13293c](https://github.com/openxrlab/xrmocap/pull/61/commits/c13293ccdd177e070d8c57867f35f7590ede90c6).